io/plugins/eu.esdihumboldt.hale.io.xls.ui/src/eu/esdihumboldt/hale/io/xls/ui/XLSInstanceExportConfigurationPage.java

```diff
@@ -155,48 +153,30 @@ public void checkStateChanged(CheckStateChangedEvent event) {
 				}
 			});
 
-			featureTypeTable.setCheckStateProvider(new ICheckStateProvider() {
-
-				@Override
-				public boolean isChecked(Object element) {
-					if (!(element instanceof TypeDefinition))
-						return false;
-					return checkboxState(element);
-				}
+			page.layout();
+			page.pack();
+		}
+	}
 
-				@Override
-				public boolean isGrayed(Object element) {
-					if (!(element instanceof TypeDefinition))
-						return false;
-					return checkboxState(element);
-				}
+	/**
+	 * @param element element to be checked
+	 * @return true if the button cannot be selected
+	 */
+	private boolean checkboxState(Object element) {
```

Comment:
Maybe rename the method to better reflect its new function (e.g. `hasData`?).
I think it would be better to change the type of `element` to `TypeDefinition` to avoid accidental wrong use.